src/dodal/beamlines/i22.py

```diff
@@ -25,10 +25,18 @@
 from dodal.utils import BeamlinePrefix, get_beamline_name
 
 BL = get_beamline_name("i22")
+IS_LAB = BL == "p38"
+LINKAM_IS_IN_LAB = False
```

Comment:
Should: Our current design for beamlines where some equipment may or may not be there is to try to initialise every device, report the failures and move on, so we should always try to connect the linkam.

Reply:
ok

Reply:
deleted that part